### DISCUSSION TITLE
start and end time parameters for jaeger data source

### DISCUSSION THREAD

**Discussion by alburthoffman** [grafana/grafana#64085] [Data Sources]
refer to https://github.com/jaegertracing/jaeger/issues/4150

The old jaeger `/api/traces/:id` does not have start and end time parameters, which makes it a global scan request.
Our data volume is super high. And it is very slow to do a global scan. 

We had discussed this with jaeger team and the time parameters would be helpful in this case. 
The changes happen in Grafana jaeger console instead of jaeger project side, as we implement the API by ourself.
And the APIs are more likely to integrate with Grafana console.

So I would like to propose adding start and end time parameters. And they are optional.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82842. Feel free to continue the discussion around this there. Thank you!
